src/parser.cc

```diff
@@ -1,17 +1,14 @@
 /*************************************************************************
- * Copyright (c) 2019-2022, NVIDIA CORPORATION. All rights reserved.
- * Modifications Copyright (c) 2019-2022 Advanced Micro Devices, Inc. All rights reserved.
- * Modifications Copyright (c) Microsoft Corporation. Licensed under the MIT License.
- *
- * See LICENSE.txt for license information
+ * Copyright (c) 2019-2023 Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE.txt for license information
  ************************************************************************/
 
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <unistd.h>
 #include <fcntl.h>
 #include <ctype.h>
-#include "parser.h"
+#include "include/parser.h"
```

Comment:
Revise makefile to remove include/ prefix

Reply:
done